charts/fpl-case-service/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: fpl-case-service
 apiVersion: v1
 home: https://github.com/hmcts/fpl-ccd-configuration
-version: 1.11.7
+version: 1.11.8
```

Comment:
should we move to version 1.12 instead? I am not sure, probably doesn't matter

Reply:
probably good idea to bump up minor version and not patch
please also update flux to point to new version